.github/workflows/ci.yaml

```diff
@@ -15,15 +15,19 @@ jobs:
   test-and-verify:
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go
-        uses: actions/setup-go@v5.1.0
-        with:
-          go-version: '1.22.2'
-
       - uses: actions/checkout@v4.2.2
         with:
           path: ${{ env.GOPATH }}/src/k8s.io/autoscaler
 
+      - name: Set up Go
+        uses: actions/setup-go@v5.5.0
+        with:
+          go-version: '1.22.2'
```

Comment:
should we update to 1.24 to match the go.mod-declared go version across the three cached projects?